### PR TITLE
Increase API Styleguide strictness

### PR DESCRIPTION
**Pull request by adamaltman**
This increases the strictness. There are a lot of operation descriptions to write and a few other issues to address in order to make it even more strict.

### CODE REVIEW

File: openapi/paths/files@{id}@download.yaml [hunk 1]
```diff
@@ -19,10 +19,12 @@ get:
           description: The number of bytes in the file.
           schema:
             type: integer
+          example: 48
         Content-Type:
           description: The MIME type of the file.
           schema:
             type: string
+          example: application/json
```
Comment (XaosSintez) [inline openapi/paths/files@{id}@download.yaml:27]: Probably better example here is something like `image/png`? I don't think we support uploading json files

File: openapi/paths/email-delivery-setting-verifications@{token}.yaml [hunk 2]
```diff
@@ -6,7 +6,7 @@ put:
   tags:
     - Email Delivery Settings
   summary: Verify an email delivery setting
-  operationId: VerifyEmailDeliverySettings
+  operationId: PutEmailDeliverySettingsVerification
```
Comment (XaosSintez) [inline openapi/paths/email-delivery-setting-verifications@{token}.yaml:9]: This should be changed on backend side too

File: openapi/paths/files@{id}@download.yaml (hunk 1 above)
Comment (adamaltman) [reply openapi/paths/files@{id}@download.yaml:27]: This is file download. Don't we use it for data exports? 

I also agree that no harm in changing the example.

File: openapi/paths/email-delivery-setting-verifications@{token}.yaml (hunk 2 above)
Comment (adamaltman) [reply openapi/paths/email-delivery-setting-verifications@{token}.yaml:9]: I changed another operationId too.

Comment (adamaltman) [reply openapi/paths/email-delivery-setting-verifications@{token}.yaml:9]: Thanks for your PR @XaosSintez

File: openapi/paths/files@{id}@download.yaml (hunk 1 above)
Comment (adamaltman) [reply openapi/paths/files@{id}@download.yaml:27]:
```suggestion
          example: image/png
```

**Review by ashkarpetin — APPROVED** [review]

**Review by dara123 — APPROVED** [review]

**Review by justnero — APPROVED** [review]

**Review by XaosSintez — APPROVED** [review]

**Review by vykhrystiuk — APPROVED** [review]

**Pull request merged**